synctl

```diff
@@ -2284,19 +2295,20 @@ class UpdateSyntheticTest(SyntheticTest):
         else:
             self.update_test_config = payload[0]
 
-    def __ensure_test_id_not_none(self, test_id):
-        if test_id is None or test_id == "":
-            print("Update Error: test id is None")
-            sys.exit(-1)
+    def invalid_update_options(self, invalid_options, items):
+        for option in invalid_options:
+            for key, value in items:
+                if option == key:
+                    if value is not None:
+                        self.exit_synctl(ERROR_CODE, f"option: --{key} not supported by synthetic tests")
```

Comment:
For this function `invalid_update_options`, it is used by both synthetic test and alert, error message changed to `f"option: --{key} not supported"` is better.
And I think we can optimize it to, you can test it.
```
def invalid_update_options(self, invalid_options, items):
    for key, value in items:
        if key in invalid_options and value is not None:
            self.exit_synctl(ERROR_CODE, f"option: --{key} is not supported")
```